app/_kong_plugins/graphql-rate-limiting-advanced/index.md

```diff
@@ -33,12 +33,232 @@ categories:
 
 search_aliases:
   - graphql-rate-limiting-advanced
----
 
-## Overview
+related_resources:
+  - text: GraphQL Proxy Cache Advanced plugin
+    url: /plugins/graphql-proxy-cache-advanced/
+---
 
-The GraphQL Rate Limiting Advanced plugin provides rate limiting for GraphQL queries. The GraphQL Rate Limiting plugin extends the [Rate Limiting Advanced](/plugins/rate-limiting-advanced/) plugin.
+The GraphQL Rate Limiting Advanced plugin provides rate limiting for GraphQL queries.
 
 Due to the nature of client-specified GraphQL queries, the same HTTP request to the same URL with the same method can vary greatly in cost depending on the semantics of the GraphQL operation in the body.
+To protect your GraphQL API, this plugin lets you analyze and assign costs to incoming GraphQL queries, then rate limit the consumer’s cost for a given time window.
+
+## Rate limiting strategies
+
+This plugin supports `cluster` and `redis` as rate limiting strategies. You can configure them using the plugin's [`config.strategy`](/plugins/graphql-rate-limiting-advanced/reference/#schema--config-strategy) parameter. If using the `cluster` strategy, all `config.redis` configuration values are ignored.
+
+This is different from the cost strategy ([`config.cost_strategy`](/plugins/graphql-rate-limiting-advanced/reference/#schema--config-cost_strategy)), which is the method that the plugin uses to determine GraphQL query costs.
+
+{:.warning}
+> A `cluster` strategy with a sync rate of `-1` should not be used in production with hybrid mode, DB-less mode, or Konnect, as it creates security risks.
+
+## Introspection endpoint
+
+The introspection endpoint is generated based on the [Gateway Service path](/gateway/entities/service/), so you must define a path in the Gateway Service itself, 
```

Comment:
```suggestion
The [introspection](https://graphql.org/learn/introspection/) endpoint is generated based on the [Gateway Service path](/gateway/entities/service/), so you must define a path in the Gateway Service itself, 
```
I *think* that link is correct, I wasn't sure what introspection was, so I looked it up and found that.